pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -45,7 +45,9 @@
 # enableLTO is a subset of the enableOptimizations flag that doesn't harm reproducibility.
 # enabling LTO on 32bit arch causes downstream packages to fail when linking
 # enabling LTO on *-darwin causes python3 to fail when linking.
-, enableLTO ? stdenv.is64bit && stdenv.isLinux
+# enabling LTO with musl fails with a linker error although it should be possible
+# as alpine is doing it: https://github.com/alpinelinux/aports/blob/a8ccb04668c7729e0f0db6c6ff5f25d7519e779b/main/python3/APKBUILD#L82
+, enableLTO ? stdenv.is64bit && stdenv.isLinux && stdenv.hostPlatform.libc == "glibc"
```

Comment:
So far the issue has only been confirmed with musl, so `!stdenv.hostPlatform.isMusl`

Reply:
Done